include/nonunity/materials/CauchyStressFromNEML2Receiver.h

```diff
@@ -0,0 +1,38 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/*                       BlackBear                              */
+/*                                                              */
+/*           (c) 2017 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#pragma once
+
+#include "CauchyStressFromNEML2UO.h"
+#include "ComputeLagrangianObjectiveStress.h"
+
+/**
+ * The "glue" material retrieves the batched output vector from a NEML2 material model and uses the
```

Comment:
```suggestion
 * This is a "glue" material that retrieves the batched output vector from a NEML2 material model and uses the
```

Reply:
Suggestion above is because it wasn't immediately obvious what class the "glue" material you referred to was.